docs/developing.md

```diff
@@ -101,7 +101,7 @@ The solidity formatting uses an npm executable named `prettier` which, for solid
 needs a plugin called `prettier-plugin-solidity`. For solidity linting, we use the
 `solhint` program.
 ```
-npm install --global --save-dev prettier prettier-plugin-solidity
+npm install --global --save-dev prettier@2.8.8 prettier-plugin-solidity
```

Comment:
I apologize. I made the same changes separately, when I was looking at the CI failures in your other PR. Let's close this PR and use https://github.com/AthanorLabs/atomic-swap/pull/502 instead. We need to fix both CI issues in order to get a PR that passes CI and can be merged.

I also opened a ticket with the plugin developers: https://github.com/prettier-solidity/prettier-plugin-solidity/issues/885